apps/evm/src/hooks/useIsFeatureEnabled/index.tsx

```diff
@@ -110,6 +110,16 @@ export const featureFlags = {
   ],
   gaslessTransactions: [ChainId.ZKSYNC_SEPOLIA, ChainId.ZKSYNC_MAINNET],
   web3DomainNames: [ChainId.BSC_MAINNET, ChainId.ETHEREUM, ChainId.ARBITRUM_ONE],
+  importPositions: [
+    ChainId.BSC_TESTNET,
+    ChainId.SEPOLIA,
+    ChainId.OPBNB_TESTNET,
+    ChainId.ARBITRUM_SEPOLIA,
+    ChainId.ZKSYNC_SEPOLIA,
+    ChainId.OPTIMISM_SEPOLIA,
+    ChainId.BASE_SEPOLIA,
+    ChainId.UNICHAIN_SEPOLIA,
+  ],
```

Comment:
nit: one thing I'd start doing in this file at is to have lists like `testnets`, `mainnets` and `allChains`, it would be easier to mantain this and other feature flags